src/mame/drivers/magic10.cpp

```diff
@@ -124,6 +124,7 @@ Both setups show different variants for components layout, memory size, NVRAM, e
 #include "cpu/m68000/m68000.h"
 #include "machine/nvram.h"
 #include "sound/okim6295.h"
+#include "machine/ticket.h"
```

Comment:
Please keep headers sorted.

Reply:
Would it be possible to make a derived state class for the machines with the hopper and ticket dispenser to avoid optional device finders, and the potential pitfalls?

I'm Sorry.I have no clue how to do this that way. My skills are on reverse engineering. Perahaps other members can take advantage on the changes suggested to make this devices working.

Anyway, thanks for your comments. I hope to be useful next time.